keps/sig-network/2681-pod-host-ip/README.md

```diff
@@ -315,7 +315,10 @@ Pod has a `status.hostIPs` field and use it in downwardAPI to expose it.
 
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
-- TBD
+- [x] Metrics
```

Comment:
Is it ok to add a count of assigned HostIPs in a kubelet?

Reply:
I'm ok with that.

[I'm not entirely sure if it's not an overkill to have a metric for this feature though...]
@dgrisonnet - do you have any thoughts?

Reply:
I share that feeling, I don't think it is useful to monitoring this feature.

Reply:
I suggest removing this metric then